api/python/indigo/__init__.py

```diff
@@ -1266,6 +1266,41 @@ def checkSalt(self):
                     return True
         return False
 
+    def stripSalt(self, inplace=False):
+        """Molecule method strips all inorganic components.
+
+        Args:
+            inplace(bool): if False - returns the copy of the molecule,
+                           without inorganic components, if True - strips
+                           inorganic components from the molecule itself.
+
```

Comment:
IMHO, it's better to design method as thin as possible: to solve just a single task.
This parameter overloads a single method with two tasks: clone and strip, while clone is already implemented. 
My proposal: leave only in-place mode. If the caller needs to operate on a copy, it can create it first.

Reply:
Other hand, might be useful "read-only" Boolean method that checks if there is any atom to be stripped. Than the caller can call it first and skip unnecessary clone, since the resulting object will be the same.

Reply:
`inplace: bool = False` is a standard pattern in Python code, so I suggest continue using it.